tests/integration_tests/_utils/server.toml

```diff
@@ -0,0 +1,3 @@
+[debug]
+# todo: remove this after v2 is enabled as the default
+enable-kafka-sink-v2 = true
```

Comment:
add some test that `enable-kafka-sink-v2=false` to test the sarama logic

Reply:
Will this break other tests? Because some tests use this arg.

Also, why don't we just enable it in the code?  The release6.6 has been created.

Reply:
> Will this break other tests? Because some tests use this arg.

I mean `./cdc server --config xxx`